lpc55_areas/src/lib.rs

```diff
@@ -512,12 +538,20 @@ impl CMPAPage {
         Ok(())
     }
 
+    pub fn get_secure_boot_cfg(&self) -> Result<SecureBootCfg> {
+        Ok(SecureBootCfg::unpack(&self.secure_boot_cfg.to_be_bytes())?)
```

Comment:
_Nothing_ in the LPC55 is big-endian. Why are these `to_be_bytes`?

Reply:
This was from @kc8apf 's initial commit in the branch, so I'm not actually sure – Rick, do you remember why this is using `to_be_bytes`?

Reply:
Never mind, there's the requested [huge block comment](https://github.com/oxidecomputer/lpc55_support/blob/master/lpc55_areas/src/lib.rs#L501-L508) right above.